gigantum/gigantum-client
# Add multi-arch build support

## Pull Request for Issue #1772

**Describe changes introduced by this PR (Bugs fixed, features added, docs updated, etc.)**:
This PR updates `gtm` to provide arm64 build support. The sass lib was also swapped out in the frontend because it was not only recently deprecated, but it also did not support arm builds.

Now there is an additional `--multi-arch` flag that will build and push an arm64 and amd64 image.

**Please check if the PR fulfills these requirements**

  - PR is set to merge to integration
  - PR title is descriptive, as this becomes the commit message on squash and merge.
  - Any related issues are added to the extended commit message during squash and merge.
  - Tests have been added (for bug fixes/features)
  - Docs have been added/updated (for bug fixes/features)

**QA Test Steps and Exit Criteria**:
- Functionally nothing should change. Verify the client test framework on both an amd64 (e.g. intel macbook) and arm64 (e.g. aws instance)
